integration-tests/gradle/src/test/java/io/quarkus/gradle/devmode/GrpcDevModeTest.java

```diff
@@ -0,0 +1,24 @@
+//package io.quarkus.gradle.devmode;
+//
+//import static org.assertj.core.api.Assertions.assertThat;
+//
+//import com.google.common.collect.ImmutableMap;
+// // TODO: to be uncommented with #10631
```

Comment:
This should be removed.

Reply:
I'd rather keep it and uncomment it in #10631, if that's okay.